src/tribol/geom/ContactPlane.hpp

```diff
@@ -30,15 +30,17 @@ class ContactPlaneManager;
  * \param [in] pair interface pair containing pair related indices
```

Comment:
I don't see where the pair is modified. The contact plane is created, but the pair is a fixed interface pair, at least for this routine. 

We can certainly consider your points (1) and (2) later on.

Reply:
If the interface pair isn't modified, then it should be `const` in the function signature.

Reply:
Unfortunately this will have a cascading effect touching many routines. While we should fix it, it is outside the scope and schedule of this PR. Will make an issue.